test/c/test_substrait_c_api.cpp

```diff
@@ -13,7 +10,6 @@ using namespace std;
 
 TEST_CASE("Test C Get and To Substrait API", "[substrait-api]") {
   DuckDB db(nullptr);
-  db.LoadExtension<duckdb::DUCKDB_EXTENSION_CLASS>();
```

Comment:
Why aren't we loading the extension?

Reply:
This symbols is not present in the code anymore. Substrait extension is part of this binary, trying to load the extension using `SubstraitExtension.Load` fails with an error `Table Function with name "get_substrait" already exists!`